src/Approvals/SFA.DAS.Approvals/InnerApi/Requests/PostBulkUploadUpdateLogRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.Approvals.InnerApi.Requests
+{
+    public class PostBulkUploadUpdateLogRequest : IPostApiRequest
+    {
+        public long? ProviderId { get; }
+        public object Data { get; set; }
+        public string PostUrl => $"api/{ProviderId}/bulkupload/update-log";
+
+        public PostBulkUploadUpdateLogRequest(long? providerId, BulkUploadUpdateLogRequest data)
+        {
+            ProviderId = providerId;
+            Data = data;
+        }
+    }
+}
```

Comment:
sorry, see previous comments about unused code

Reply:
hi paul, removed this as part of removing all Update endpoint